ypathgen/pathgen.go

```diff
@@ -485,6 +486,8 @@ type NodeData struct {
 	YANGTypeName string
 	// YANGPath is the schema path of the YANG node.
 	YANGPath string
+	// GoPackage is the Go package name of a schema node.
```

Comment:
`NodeData` represents both GoStruct and PathStruct information. So maybe something more specific:

"GoPathPackageName is the Go package name containing the generated PathStruct for the schema node."

Reply:
done